### PR TITLE
Inline edge peclet blending again

### PR DESCRIPTION
**Pull request by psakievich**
**Pull-request type:**
- [x] Bug fix 
- [ ] Documentation update
- [ ] Feature enhancement

## Description of the pull-request

Revert specific changes for Peclet comp from #725 in response to #768 

## Checklist

**NOTE** The checklist below is a suggestion and not mandatory. You don't have
to _check all boxes_ to submit a pull request. However, please add a brief
explanation in your pull request summary explaining the omission for the benefit
of reviewers and developers.

*All pull requests*
- [x] Builds successfully (*must test on at least one system/compiler combination*)
  - Operating systems 
    - [ ] Linux
    - [x] MacOS
  - Compilers 
    - [ ] GCC
    - [x] LLVM/Clang
    - [ ] Intel compilers
    - [ ] NVIDIA CUDA
- [x] Compiles without warnings
- [x] Passes all unit tests
- [ ] Passes all regression tests
- [ ] Documentation builds without errors

*For new code updates*
- [ ] Documentation updates - additions to user, theory, and verification manuals
- [ ] New unit tests providing coverage for new code, bug fixes etc.
- [ ] New regression tests exercising the new code

### CODE REVIEW

**Comment by psakievich** [comment]
@sayerhs I can't seem to link #768  to this PR due to permissions. Would you do it for me?  I will wait for confirmation that this fixes the issues from you and @marchdf before attempting to merge.

**Comment by marchdf** [comment]
@psakievich This fixed the issue I was seeing. I now have this log file output:
```
   Nalu-Wind Version: v1.2.0
   Nalu-Wind GIT Commit SHA: 698b69d668c8285773c5a418283cc14146f90977
   Trilinos Version: 13.1-gc23783ec3ea

[...]

Time Step Count: 1 Current Time: 0.001
 dtN: 0.001 dtNm1: 0.001 gammas: 1 -1 0
 Max Courant: 4508.09 Max Reynolds: 24694.4 (realm_1)
   Realm Nonlinear Iteration: 1/1

realm_1::advance_time_step()
NLI    Name           Linear Iter      Linear Res     NLinear Res    Scaled NLR
---    ----           -----------      ----------     -----------    ----------
1/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS            9       0.000234933      0.0263994             1
        ContinuityEQS         14        0.00668293        0.48266             1
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   4        0.00385242       0.179234             1
        SpecDissRateEQS        9           586.474        50798.9             1
2/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS           10        0.00294428       0.191212       7.24306
        ContinuityEQS          9          0.195563        14.1594       29.3362
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   5       0.000170552      0.0106145     0.0592218
        SpecDissRateEQS        1       1.37708e+06    7.10402e+08       13984.6
3/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS           10        0.00251245      0.0990267        3.7511
        ContinuityEQS          9          0.135597        8.99556       18.6375
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   6       3.84558e-05     0.00213096     0.0118893
        SpecDissRateEQS        1            432282     1.7878e+09       35193.6
4/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS           10        0.00139753      0.0616717        2.3361
        ContinuityEQS          8          0.130581        6.03221       12.4978
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   8       6.41463e-06    0.000490944    0.00273913
        SpecDissRateEQS        1           79487.4     2.6693e+10        525464
Mass Balance Review:
Density accumulation: 0
Integrated inflow:    -33809.99999999998
Integrated open:      33810.04149369935
Total mass closure:   0.0414936993693118
Mean System Norm: 5338599906.149566 1 0.001
WallClockTime: 1 Pre: 0.00133109 NLI: 16.9942 Post: 0.00799918 Total: 17.0035
*******************************************************
Time Step Count: 2 Current Time: 0.002
 dtN: 0.001 dtNm1: 0.001 gammas: 1.5 -2 0.5
 Max Courant: 4959.01 Max Reynolds: 37470.2 (realm_1)
   Realm Nonlinear Iteration: 1/1

realm_1::advance_time_step()
NLI    Name           Linear Iter      Linear Res     NLinear Res    Scaled NLR
---    ----           -----------      ----------     -----------    ----------
1/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS            9         0.0031364       0.177682             1
        ContinuityEQS         11         0.0931651        6.19765             1
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   4        0.00122793       0.256523             1
        SpecDissRateEQS        1             22904    1.43046e+11             1
2/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS            9        0.00144713       0.126069      0.709522
        ContinuityEQS         10         0.0830741        4.36541      0.704365
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   5       0.000180926      0.0109962     0.0428664
        SpecDissRateEQS        1           9657.35    1.88317e+11       1.31648
3/4    Equation System Iteration
 1/1      myLowMach
        MomentumEQS            9       0.000829767      0.0753129      0.423864
        ContinuityEQS         11         0.0514613        3.12134      0.503632
 1/1ShearStressTransportWrap
        TurbKineticEnergyEQS   6       2.32729e-05     0.00155155    0.00604838
        SpecDissRateEQS        1           5857.37    1.52904e+11       1.06891
```

**Review by marchdf — APPROVED** [review]

**Pull request merged**
